src/panels/lovelace/hui-root.ts

```diff
@@ -728,7 +730,7 @@ class HUIRoot extends LitElement {
     window.history.replaceState(
       null,
       "",
-      constructUrlCurrentHref(createSearchParam({ edit: "1" }))
+      constructUrlCurrentHref(addSearchParam({ edit: "1" }))
```

Comment:
this really just is `window.location.pathname + "?" + addSearchParam({ edit: "1" })`

Not really helper worthy?

Reply:
I wanted to prevent "dangling" question marks at the end of the URL to keep the URLs "clean". That's the main purpose of the helper.